### DISCUSSION TITLE
Add pull_request_review_thread.created & pull_request_review_thread.deleted webhook events

### DISCUSSION THREAD

**Discussion by nggonzalez** [community/community#41043] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

## Context

Currently, webhooks for `pull_request_review_thread`s are limited to two events, `resolved` and `unresolved`. 

For experiences that revolved around review threads, however, this event set is too restrictive and makes it impossible to know when:
- Review threads are created
- Review threads are deleted
- When comments are added to a review thread
 
In addition, the `pull_request_review_comment` events do not include a `pullRequestReviewThreadId`, so there is no workaround using the comment events because there is no thread id present.

I've also explored getting the thread ids via the client side, but this is (1) unreliable and (2) the thread ids used by the resolve / unresolve buttons are integers rather than the ids exposed by the webhooks events.

## Feature Requests

1. Add a `pull_request_review_thread.created` event that is sent when the first comment in a thread is created and returns a similar payload to the existing events `pull_request_review_thread` events. 
2. Add the thread's integer id to the exposed `thread` event data
3. Update the `pull_request_review_comment` to contain a reference to the review thread's node and integer ids
4. Add a `pull_request_review_thread.deleted` event that is sent when all comments in a thread are deleted and returns a similar payload to the existing events `pull_request_review_thread` events.

**Comment by ossanna16** [comment]
Hi there @nggonzalez and welcome to our community! Thank you for the feedback!
